backend/src/scraper/walmart_scraper.py

```diff
@@ -89,20 +89,28 @@ def product_reviews(product_id: str) -> Dict:
             }
         else:
             top_positive = results.get("top_positive", {})
+            if top_positive.get("text") is not None:
+                top_positive_details = {
+                    "title": top_positive.get("title"),
+                    "text": top_positive.get("text"),
+                    "rating": top_positive.get("rating"),
+                }
+            else:
+                top_positive_details = {}
```

Comment:
Should this if-statement logic be added for best buy reviews too?

Reply:
No, because we only calculate sentiment when there are more than 0 reviews. There's no chance that null will be returned by the sentiment method.

Reply:
Makes sense, thanks for clarifying!